### PR TITLE
Bookstore: Use Redux in React components.

### PR DESCRIPTION
**Pull request by mrprotocoll**
### The above updates were made

- Add remove book feature using redux
- Add display book feature using redux
- Add add book feature using redux
- Add check status click feature

### CODE REVIEW

File: src/routes/Categories.js [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
+import { useDispatch } from 'react-redux';
+import { checkStatus } from 'redux/categories/categoriesSlice';
+
+const Categories = () => {
+  const dispatch = useDispatch();
+
+  return (
+    <div className="container">
+      <button type="button" onClick={() => dispatch(checkStatus())}>Check status</button>
+    </div>
```
Comment (CollinsTatang) [inline src/routes/Categories.js]: - [ ] Please your `Check Status` button does not dispatches the Under Construction action, try to fix that.

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @mrprotocoll, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/routes/Categories.js (hunk 1 above)
Comment (mrprotocoll) [reply src/routes/Categories.js]: Hi @CollinsTatang ,
Thanks for the feedback. Could you please help me with more details on this, I really do not understand what I am supposed to fix. Thank you

Comment (CollinsTatang) [reply src/routes/Categories.js]: Can we have a zoon call so that I can better explain to you

Comment (mrprotocoll) [reply src/routes/Categories.js]: yes, thank you

Comment (CollinsTatang) [reply src/routes/Categories.js]: Hello @mrprotocoll 
When a User Click the `Check Status` button under categories nothing happened
It supposes to display `Under Construction `

Comment (CollinsTatang) [reply src/routes/Categories.js]: > yes, thank you

Send a zoom call link

Comment (mrprotocoll) [reply src/routes/Categories.js]: Alright, Thank you @CollinsTatang , I will make the changes and request a new review

**Review by binhussen — APPROVED** [review]
Hi @mrprotocoll , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- used react-redux ✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**